qulice-findbugs/src/main/java/com/qulice/findbugs/Wrap.java

```diff
@@ -76,6 +75,13 @@ public static void main(final String... args) {
     /**
      * Run it.
      * @param args Arguments
+     * @todo #706:30min Temporary disable BED_BOGUS_EXCEPTION_DECLARATION check
```

Comment:
@dskalenko `Temporary disable BED_BOGUS_EXCEPTION_DECLARATION check` - but this is what you did, puzzle is only about next task